create_infotexts.py

```diff
@@ -435,6 +437,14 @@ def process_region_addition_to_description(region_str, country_str):
 
         return region_addition
 
+    def add_catch_all_category(self, item):
+        """"
+        Check if there are any content cats added and add generic content category to image.
+        
+        :return: None (populates self.content_cats)
```

Comment:
Don't describe this as a return. You can just have `populates self.content_cats` as part of the docstring  (on it's own line)